src/dpdkcap.c

```diff
@@ -274,11 +275,19 @@ static int port_init(
   uint16_t q;
 
   /* Check if the port id is valid */
+#if RTE_VERSION >= RTE_VERSION_NUM(18,11,3,16)
```

Comment:
Maybe it would be simpler to declare a variable, to limit the amount of duplicated code. Something like:
```
#if RTE_VERSION >= RTE_VERSION_NUM(18,11,3,16)
int dev_count = rte_eth_dev_count_avail()-1;
#else
int dev_count = rte_eth_dev_count()-1;
#endif
  if(rte_eth_dev_is_valid_port(port)==0) {
    RTE_LOG(ERR, DPDKCAP, "Port identifier %d out of range (0 to %d) or not"\
       " attached.\n", port,  dev_count);
    return -EINVAL;
  }
  }
```

Reply:
OK.
It cannot pass the build. So I modified travis.yaml